applications/nrf5340_audio/dfu/conf/Kconfig.dfu

```diff
@@ -111,6 +111,18 @@ config MCUBOOT_IMAGE_VERSION
 	string
 	default "0.0.0+0"
 
+config BT_DFU_DEVICE_NAME
+	string
+	default "NRF5340_AUDIO_HAPPY_DFU"
+
+config LOG_DFU_ENTRY_LEVEL
+	int "Log level for dfu_entry.c"
```

Comment:
Please use a proper name, and not the C-file here:
```
	int "Log level for <more descriptive name>"
```

Reply:
Change to "Log level for DFU entry module".